docs/api-reference/uiSchema.md

```diff
@@ -64,7 +109,33 @@ Will result in:
 </div>
 ```
 
-## autofocus
+#### BREAKING CHANGE in version 5
+In versions previous to 5, `classNames` was the only property that did not require the `ui:` prefix. 
+Additionally, it did not support being added into the `ui:options` object.
+This was fixed in version 5 to be consistent with all the other properties in this section.
+If you were using `classNames` as follows, simply add the `ui:` prefix to it:
+
+```jsx
+const uiSchema = {
+  title: {
+    "classNames": "my class"
```

Comment:
I think this is a big breaking change and to make it less painful to users to upgrade, what about just showing a warning instead of ignoring classNames completely? For example, I use rjsf in an upstream library where I store tons of form schemas / uiSchemas in a database. It's not feasible for me to rename classNames in all of them to ui:classNames.

The fundamental issue is that this breaking change is not as easy as just changing code, but could potentially require large data migrations, which is much more cumbersome. I wonder if many others would have a similar issue which would preclude them from upgrading to v5.

Reply:
Do those same `schema`s you have tucked away also use `enumNames` because those are being removed too :(